src/components/DeeplinkWrapper/index.website.js

```diff
@@ -59,8 +63,9 @@ class DeeplinkWrapper extends PureComponent {
         if (prevProps.session.shortLivedAuthToken || !this.props.session.shortLivedAuthToken) {
```

Comment:
Are we not handling the errors we were handling before anymore?

Reply:
Can you remind me what errors we were handling and why they are not necessary anymore? I looked at the linked PR but I'm not seeing anything about errors or error handling (maybe I missed it).